src/message/mimebody.rs

```diff
@@ -330,6 +344,13 @@ impl MultiPart {
         MultiPart::builder().kind(MultiPartKind::Signed { protocol, micalg })
     }
 
+    /// Alias for HTML and plain text versions of an email
+    pub fn alternative_html_plain<T: IntoBody>(html: T, plain: T) -> Self {
+        Self::alternative()
+            .singlepart(SinglePart::html(html))
+            .singlepart(SinglePart::plain(plain))
+    }
```

Comment:
Is there any reason for having `html` before `text`? Usually I see them the other way around.

Reply:
Mainly because text is often a fallback, but I'll switch if it seems more usual